packages/stryker-karma-runner/src/TestHooksMiddleware.ts

```diff
@@ -0,0 +1,25 @@
+import { RequestHandler } from 'express';
+import * as url from 'url';
+
+export const TEST_HOOKS_FILE_NAME = '__test_hooks_for_stryker__.js';
+
+export default class TestHooksMiddleware {
+
+  public currentTestHooks: string = '';
+
+  public handler(): RequestHandler {
+    const self = this;
```

Comment:
Is this really needed? We're using a arrow function so the context of `this` should still be the same.